Wikipedia/Code/DiffHeaderViewModels.swift

```diff
@@ -80,21 +80,43 @@ final class DiffHeaderViewModel: Themeable {
         apply(theme: theme)
     }
     
-    static func generateTitleViewModelForCompare(articleTitle: String, counts: (revision: Int, user: Int)?) -> DiffHeaderTitleViewModel {
-        
-         let heading = WMFLocalizedString("diff-compare-header-heading", value: "Compare Revisions", comment: "Heading label in header when comparing two revisions.")
-        
-        if let counts = counts {
-           
-            let subtitleFormat = WMFLocalizedString("diff-compare-header-subtitle-format", value:"%1$@ by %2$@ not shown", comment:"Subtitle label in header when comparing two revisions. %1$@ is replaced with the number of intermediate revisions between chosen revisions to compare, and %2$@ is replaced by the number of editors that made those intermediate revisions.")
```

Comment:
In some languages, the connecting "by" will be preceded by an adjective whose form depends on the number of revisions. If the "by" is translated in isolation, it'll be incorrect (similar to https://phabricator.wikimedia.org/T231566)